bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GroupItem.java

```diff
@@ -27,6 +28,8 @@
 
     private final Logger logger = LoggerFactory.getLogger(GroupItem.class);
 
+    private State oldGroupState = UnDefType.NULL;

```

Comment:
what is this field good for? Why do we have to keep an "old" state and what does "old" exactly mean?